src/Integration.UnitTests/Binding/CSharpVBProjectBinderTests.cs

```diff
@@ -175,14 +191,14 @@ public void IsBound_ProjectHasOneRuleset_CantLoadProjectRulesetFile_False()
                 .Setup(x => x.TryGetProjectRuleSetFilePath(projectMock, ruleSetDeclaration, out filePath))
                 .Returns(false);
 
-            var result = testSubject.IsBound(bindingConfiguration, projectMock);
-            result.Should().BeFalse();
+            var result = testSubject.IsBindingRequired(bindingConfiguration, projectMock);
+            result.Should().BeTrue();
```

Comment:
Perhaps verify the expected mocks were called, similar to what you'd done in the earlier tests?